reana_client/cli/secrets.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2019 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA client workflow related commands."""
+import logging
+import sys
+import traceback
+
+import click
+from reana_client.api.client import add_secrets, delete_secrets, \
+    list_secrets, current_rs_api_client
+from reana_client.cli.utils import add_access_token_options
+from reana_client.config import ERROR_MESSAGES
+from reana_commons.errors import MissingAPIClientConfiguration, \
+    REANASecretAlreadyExists, REANASecretDoesNotExist
+
+from reana_commons.utils import click_table_printer
+
+from reana_client.utils import parse_secret_from_path, \
+    parse_secret_from_literal
+
+
+@click.group(help='Secrets management commands')
+@click.pass_context
+def secrets_group(ctx):
+    """Top level wrapper for secrets management."""
+    logging.debug(ctx.info_name)
+
+
+@secrets_group.command()
+@click.option(
+    '--from-literal',
+    multiple=True,
+    help='Secrets to be uploaded from literal string.'
+         'e.g. PASSWORD=password123')
+@click.option(
+    '--from-file',
+    multiple=True,
+    help='Secret to be uploaded from file.')
+@click.option(
+    '--overwrite',
+    is_flag=True,
+    default=False,
+    help='Overwrite the secret if already present')
+@add_access_token_options
+def secrets_add(from_literal, from_file, overwrite, access_token):  # noqa: D301
+    """Add secrets from literal string or from file.
+
+    Examples: \n
+    \t $ reana-client secrets-add --from-literal PASSWORD=password \n
+    \t $ reana-client secrets-add --from-file ~/.keytab \n
+    \t $ reana-client secrets-add --from-literal USER=reanauser \n
+    \t                            --from-literal PASSWORD=password \n
+    \t                            --from-file ~/.keytab
+    """
+    if not access_token:
+        click.echo(
+            click.style(ERROR_MESSAGES['missing_access_token'], fg='red'),
+            err=True)
+        sys.exit(1)
+
+    secrets = {}
+    for literal in from_literal:
+        secret = parse_secret_from_literal(literal)
+        secrets.update(secret)
+    for path in from_file:
+        secret = parse_secret_from_path(path)
+        secrets.update(secret)
+    try:
+        add_secrets(secrets, overwrite, access_token)
+    except REANASecretAlreadyExists as e:
+        logging.debug(traceback.format_exc())
+        logging.debug(str(e))
```

Comment:
```suggestion
        logging.debug(str(e), exc_info=True)
```

We can use this keyword parameter so we do not have to explicitly log the `traceback` :)